src/python/grpcio_tests/tests_py3_only/interop/xds_interop_client.py

```diff
@@ -18,12 +18,23 @@
 import datetime
 import logging
 import signal
-import sys
 import threading
 import time
-from typing import DefaultDict, Dict, List, Mapping, Sequence, Set, Tuple
+from typing import (
+    Any,
+    DefaultDict,
+    Dict,
+    Iterable,
+    List,
+    Mapping,
+    Sequence,
+    Set,
+    Tuple,
+    Union,
+)
```

Comment:
`Any` isn't used anymore
```suggestion
from typing import DefaultDict, Dict, List, Mapping, Sequence, Set, Tuple
```